backend/de.metas.business/src/main/java/de/metas/tax/api/impl/TaxDAO.java

```diff
@@ -213,4 +190,15 @@ public ITranslatableString getTaxCategoryNameById(@Nullable final TaxCategoryId
 
 		return Optional.ofNullable(taxCategoryId);
 	}
+
+	@Override
+	public Percent getRateById(@NonNull final TaxId taxId)
+	{
+		final I_C_Tax tax = queryBL.createQueryBuilder(I_C_Tax.class)
```

Comment:
now that i am looking, why not using `getTaxById`... even overriding with a method which takes `TaxId` as param?

Reply:
done